data/store/mongo/mongo_bucket_store_client.go

```diff
@@ -96,7 +105,32 @@ func (c *MongoBucketStoreClient) UpsertMany(ctx context.Context, userId *string,
 	return nil
 }
 
-func buildUpdateOneModel(dataType string, sample schema.ISample, userId *string, ts string, creationTimestamp time.Time) ([]mongo.WriteModel, error) {
+func buildCbgUpdateOneModel(sample schema.ISample, userId *string, ts string, creationTimestamp time.Time) ([]mongo.WriteModel, error) {
```

Comment:
Could we rename `ts` by `date`? Or add a documentation on the function to understand what ts is?